app/src/main/res/values/theme_custom_light.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="CustomTheme_Light" parent="AppTheme.NoActionBar">
+        <item name="android:textColorSecondary">@color/dark_gray</item>
+        <item name="android:textColorTertiary">@color/gray</item>
+        <item name="android:windowAnimationStyle">@style/WindowAnimationTransition</item>
```

Comment:
Mainly mimic the skeleton theme, this is because most of the color swapping and changes gets applied dynamically.